nixos/lib/make-options-doc/options-to-docbook.xsl

```diff
@@ -189,7 +189,7 @@
   </xsl:template>
 
 
-  <xsl:template match="derivation">
+  <xsl:template match="attrs[attr[@name = '_type' and string[@value = 'derivation']]]">
```

Comment:
What's this change for?

Reply:
nix xml makes a distinction between derivations and attrsets, but json can't. to get around this we instead format derivations as a `_type` special value, like options and examples are handled already

Reply:
Makes sense!